src/CLICommand/parameters_helper.test.ts

```diff
@@ -268,7 +272,9 @@ describe('ParametersHelper class', () => {
 		// it('returns the address of the wallet when a valid --seed-phrase option is provided', () => {
```

Comment:
What's up with these tests?

Reply:
those are the seed phrase tests we commented out

Reply:
Sorry I got lazy. I see that they're the slow --seed-phrase tests